content/docs/examples/multicluster/icp/index.md

```diff
@@ -4,13 +4,13 @@ description: Example multicluster mesh over two IBM Cloud Private clusters.
 weight: 70
 keywords: [kubernetes,multicluster]
 aliases:
-    - /docs/examples/multicluster/icp/
+    - /docs/tasks/multicluster/icp/
 ---
 
 This example demonstrates how to setup network connectivity between two
 [IBM Cloud Private](https://www.ibm.com/cloud/private) clusters
 and then compose them into a multicluster mesh using a
-[single control plane with VPN connectivity](/docs/concepts/multicluster-deployments/#single-control-plane-with-vpn-connectivity)
+[single-network shared control plane](/docs/concepts/multicluster-deployments/#single-network-shared-control-plane)
```

Comment:
as mentioned prior, single-network needs refinement imo.  This could work as a label, however, the context of the label should be set somewhere in the documentation more completely.

Reply:
Maybe single-network and multi-network should be glossary entries that we reference.
^^^ @adammil2000